docs/guidelines/index.md

```diff
@@ -0,0 +1,176 @@
+
+If you have been selected, we extend our warmest congratulations! The community bonding period starts within two days after the official announcement, so get ready to dive into the exciting world of open-source development with FOSSology.
+
+## Community Bonding
+
+As soon as the GSoC contributors are accepted, the community bonding period starts. During this period, contributors are not expected to be coding immediately. Instead, they are expected to prepare to code.
+A thriving community bonding period leads to successful coding, so make the most of it!
+
+Use the community bonding to:
+
+- Define the communication channels with your mentors.
+- Schedule weekly meetings with your mentors.
+- Continue to discuss and plan the project with the community and the mentors.
+- Set up your computer and development environment to work on the project (see [Useful links](#useful-links)).
+
+## Coding Periods
+
+During the coding periods, GSoC contributors are expected to:
+
+- Work on the GSoC project.
+- Follow the [Code Style Best Practices](https://github.com/fossology/fossology/wiki/Coding-Style).
+- Write a short summary of the work done each week, published to FOSSolgy GSoC tracket. It is also advisable to publish in a personal blog as well.
+    - ``It’s okay to say things like <this> and <that> were challenging because of <reason>.``
+- Say something when you are stuck, lost in the code, confused about the objectives, etc.
+- Notify in advance when taking time off and make use of weekends to rest and avoid burnout.
+- Be present online and proactively reach out to the community if required.
+
+## Evaluations
+
+At the end of each coding period, GSoC contributors are expected to:
+
+- Publish a blog post summarizing your status on your blog or on Medium.
+- Announce the blog post on the [Slack channel](https://join.slack.com/t/fossology/shared_invite/enQtNzI0OTEzMTk0MjYzLTYyZWQxNDc0N2JiZGU2YmI3YmI1NjE4NDVjOGYxMTVjNGY3Y2MzZmM1OGZmMWI5NTRjMzJlNjExZGU2N2I5NGY).
+
+Past years' blog posts may inspire you. Here are some links:
+
+https://medium.com/@rohit-pandey/gsoc-2022-final-report-bf1e34440bce
+
+https://medium.com/@shruti-ag/gsoc21-with-fossology-fd97cd2a3924
+
+## Post-GSoC
+
+Congratulations on reaching the end of GSoC! You've accomplished so much, and the journey has only just begun. Once the final results are announced by Google, it's not the end of the road for you. Here are a few options for you to consider:
+
+- Keep the momentum going and continue to develop your project within the FOSSology community.
+- Apply again next year and take on another exciting challenge.
+- Share your knowledge and experience by becoming a mentor in Google Summer of Code for the next year.
```

Comment:
- Helping us keeping the documentations updated.
- Give presentations/talks about your work during GSoC.